### ISSUE TITLE
Craft 4 to 5 migration fails

### ISSUE THREAD

**Issue by johnrich85** [solspace/craft3-freeform#1644]
### What happened?

The `m230101_100010_FF4to5_MigrateForms` fails. It looks like i've got some long form names, and when it tries to create the new table it exceeds Mysqls limit of 64 characters.

### Errors and Stack Trace (if available)

```shell
> rename table {{%tmp_795ef}} to {{%freeform_submissions_examplelongformhandlewithreallylongmanysegmentsinhere_6}} ...Exception: SQLSTATE[42000]: Syntax error or access violation: 1059 Identifier name 'freeform_submissions_examplelongformhandlewithreallylongmanysegmentsinhere_6' is too long
```

### How can we reproduce this?

1. Create a form with a long name on Craft 4
2. Upgrade to craft 5

### Freeform Edition

Pro

### Freeform Version

5.7.2

### Craft Version

5.5.1.1

### When did this issue start?

After upgrading from older Craft version

### Previous Freeform Version

4.x

**Comment by kjmartens** [comment]
Sorry for the trouble you're experiencing @johnrich85.

If you rename the form handle to be shorter in Freeform 4, does it resolve the issue for you? That might be the fastest way for you to move along with the upgrade while we check into this on our end and make a fix to the migration. 🙂

**Comment by kjmartens** [comment]
This should be resolved in Freeform 5.7.4+ 🙂
